documentation/tutorials/sensitivity/PEcAn_sensitivity_tutorial_v1.0.Rmd

```diff
@@ -99,48 +105,41 @@ runids
 ```{r,eval=FALSE}
-arun <- read.output(runids[1],paste("~/output",RunDir,"out",runids[1],sep="/"),2006,2006,c("time","NEE"))
-plot(arun$time,arun$NEE)
+arun <- read.output(runids[1], paste("~/output", RunDir, "out", runids[1], sep="/"), start.year= 2004, end.year= 2004,"NEE", dataframe = TRUE)
+plot(arun$posix,arun$NEE)
 ```
 
 ####B. Now let’s bring in the actual observations
-Recall reading Ameriflux NEE in the earlier tutorial. This file was downloaded when we chose to get the drivers for SIPNET from Ameriflux. If it’s already in your workspace, you can skip this step. Otherwise, let's do it again.
-
-Navigate to the *~/output/dbfiles* folder. Read the Ameriflux NetCDF (.nc) file in  *~/output/dbfiles/Ameriflux_site-0-722/*  here *0-722* is the database site ID, a folder that was created when you selected the site and PEcAn downloaded the Ameriflux NetCDF (.nc) file from the Ameriflux website. The *0-722* may be a different number in your system depending on the site you chose and the database contents. Also note *US-NR1*, which refers to the Fluxnet site identifier for Niwot Ridge and the *2006* for the year.
+Recall reading Ameriflux NEE in the earlier tutorial. Read in an Ameriflux csv file, stored in the bety database. Here *722* is the database site ID which allows bety to assign the correct location of the data. The *722* may be a different number in your system depending on the site you chose and the database contents. Also note *US-NR1*, which refers to the Fluxnet site identifier for Niwot Ridge.
 ```{r,eval=FALSE}
-library(PEcAn.assim.batch)
-obs <- load.L2Ameriflux.cf("~/output/dbfiles/Ameriflux_site_0-772/US-NR1.2006.nc")
-names(obs)
-```
+File_Path<-"/fs/data3/tmccabe/AMF_US-NR1_BASE_HH_9-1.csv" #This is a path to where the observations are stored. 
+File_format<-PEcAn.DB::query.format.vars(bety = bety, format.id = 5000000002) #This matches the file with a premade "format" or a template that describes how the information in the file is organized
+site<-PEcAn.DB::query.site(site.id = 772,bety$con)  
 
-Change all bad values from -9999 to not a number (NA), and plot it to see if it make sense
-```{r,eval=FALSE}
-obs[obs==-9999]=NA
-plot(obs$NEE)
-```
+obs<-PEcAn.benchmark::load_data(data.path = File_Path, format= File_format, time.row = File_format$time.row,  site = site, start_year = 2004, end_year = 2004) 
 
-To get something we can compare to, we also need to only retain good NEE. At the least, a u* filter is required. Let’s apply a simple one
-```{r,eval=FALSE}
-niwotnee <- obs$NEE
-niwotustar <- obs$UST
-niwotnee[niwotustar<0.2]=NA
+names(obs) #Notice that there are several time columns ("$year", "$datetime", "$posix") for most plotting, "$posix" will be the best time axis
+
+annualNEEobs<-aggregate(obs$NEE, by= list(month(obs$posix)), simplify=TRUE, FUN =mean, na.rm= TRUE)
```

Comment:
Ok, but if you wanted to show a user how to do a different data alignment that should be part of the visualization tutorial -- that might be a useful addition there (e.g. imagine the hypothetical where we were provided monthly data, here's how we'd handle that with align_data), but it's not what we want for this tutorial.  Also, if you do aggregate flux data up you really need to use the gap-filled data, otherwise the means will be biased.